pkg/apiclient/workflowtemplate/workflowtemplate.go

```diff
@@ -0,0 +1,3 @@
+package workflowtemplate
+
+//go:generate mockery -name WorkflowTemplateServiceClient
```

Comment:
Could you explain this file? Just to generate the mocks?

Reply:
Yes 😄  Using this mock in one of the tests I've added.